### ISSUE TITLE
Building under io.js

### ISSUE THREAD

**Issue by MikeCostello** [schamane/node-syslog#40]
Tried building on io.js v1.5.1 and ```node-gyp rebuild``` fails.

Possibly maintain node.js/io.js support with ```nan```?

npm-debug.log
```
0 info it worked if it ends with ok
1 verbose cli [ '/Users/mikecostello/.nvm/versions/io.js/v1.5.1/bin/iojs',
1 verbose cli   '/Users/mikecostello/.nvm/versions/io.js/v1.5.1/bin/npm',
1 verbose cli   'install',
1 verbose cli   'schamane/node-syslog' ]
2 info using npm@2.7.0
3 info using node@v1.5.1
4 silly cache add args [ 'schamane/node-syslog', null ]
5 verbose cache add spec schamane/node-syslog
6 silly cache add parsed spec { raw: 'schamane/node-syslog',
6 silly cache add   scope: null,
6 silly cache add   name: null,
6 silly cache add   rawSpec: 'schamane/node-syslog',
6 silly cache add   spec: 'schamane/node-syslog',
6 silly cache add   type: 'github' }
7 info maybeGithub Attempting schamane/node-syslog from git://github.com/schamane/node-syslog
8 verbose addRemoteGit u="git://github.com/schamane/node-syslog" silent=true
9 silly addRemoteGit normalized { url: 'git://github.com/schamane/node-syslog',
9 silly addRemoteGit   branch: 'master' }
10 silly addRemoteGit v git-github-com-schamane-node-syslog-654a11bb
11 verbose addRemoteGit /Users/mikecostello/.npm/_git-remotes/git-github-com-schamane-node-syslog-654a11bb not in flight; cloning
12 info git [ 'config', '--get', 'remote.origin.url' ]
13 verbose git remote.origin.url git://github.com/schamane/node-syslog
14 info git [ 'fetch', '-a', 'origin' ]
15 verbose git fetch -a origin (git://github.com/schamane/node-syslog)
16 info git [ 'rev-list', '-n1', 'master' ]
17 verbose git rev-list -n1 master 30d445559617d439714418cdc1e847efbb49c5a0
18 verbose resolved git url git://github.com/schamane/node-syslog#30d445559617d439714418cdc1e847efbb49c5a0
19 info git [ 'clone',
19 info git   '/Users/mikecostello/.npm/_git-remotes/git-github-com-schamane-node-syslog-654a11bb',
19 info git   '/var/folders/1r/xflxmdns1wg06mf31wjwmp8r0000gn/T/npm-31126-ac5a203b/git-cache-a038274f501d/30d445559617d439714418cdc1e847efbb49c5a0' ]
20 verbose git clone from /Users/mikecostello/.npm/_git-remotes/git-github-com-schamane-node-syslog-654a11bb
21 verbose git clone Cloning into '/var/folders/1r/xflxmdns1wg06mf31wjwmp8r0000gn/T/npm-31126-ac5a203b/git-cache-a038274f501d/30d445559617d439714418cdc1e847efbb49c5a0'...
21 verbose git clone done.
22 info git [ 'checkout', '30d445559617d439714418cdc1e847efbb49c5a0' ]
23 verbose git checkout Note: checking out '30d445559617d439714418cdc1e847efbb49c5a0'.
23 verbose git checkout
23 verbose git checkout You are in 'detached HEAD' state. You can look around, make experimental
23 verbose git checkout changes and commit them, and you can discard any commits you make in this
23 verbose git checkout state without impacting any branches by performing another checkout.
23 verbose git checkout
23 verbose git checkout If you want to create a new branch to retain commits you create, you may
23 verbose git checkout do so (now or later) by using -b with the checkout command again. Example:
23 verbose git checkout
23 verbose git checkout   git checkout -b new_branch_name
23 verbose git checkout
23 verbose git checkout HEAD is now at 30d4455... Fix syntax error in 0.12 branch of syslog.cc
24 verbose addLocalDirectory /Users/mikecostello/.npm/node-syslog/1.1.7/package.tgz not in flight; packing
25 verbose tar pack [ '/Users/mikecostello/.npm/node-syslog/1.1.7/package.tgz',
25 verbose tar pack   '/var/folders/1r/xflxmdns1wg06mf31wjwmp8r0000gn/T/npm-31126-ac5a203b/git-cache-a038274f501d/30d445559617d439714418cdc1e847efbb49c5a0' ]
26 verbose tarball /Users/mikecostello/.npm/node-syslog/1.1.7/package.tgz
27 verbose folder /var/folders/1r/xflxmdns1wg06mf31wjwmp8r0000gn/T/npm-31126-ac5a203b/git-cache-a038274f501d/30d445559617d439714418cdc1e847efbb49c5a0
28 verbose addLocalTarball adding from inside cache /Users/mikecostello/.npm/node-syslog/1.1.7/package.tgz
29 silly cache afterAdd node-syslog@1.1.7
30 verbose afterAdd /Users/mikecostello/.npm/node-syslog/1.1.7/package/package.json not in flight; writing
31 verbose afterAdd /Users/mikecostello/.npm/node-syslog/1.1.7/package/package.json written
32 silly install resolved [ { name: 'node-syslog',
32 silly install resolved     version: '1.1.7',
32 silly install resolved     main: './node-syslog',
32 silly install resolved     engines: { node: '>=0.8.0' },
32 silly install resolved     os: [ '!win32' ],
32 silly install resolved     scripts: { test: 'node test.js', install: 'node-gyp rebuild' },
32 silly install resolved     description: 'Node module to support sending messages to syslog daemon',
32 silly install resolved     author: { name: 'Nazar Kulyk', email: 'nasar.kulyk@googlemail.com' },
32 silly install resolved     homepage: 'http://github.com/schamane/node-syslog',
32 silly install resolved     repository: { type: 'git', url: 'git@github.com:schamane/node-syslog.git' },
32 silly install resolved     bugs: { url: 'https://github.com/schamane/node-syslog/issues' },
32 silly install resolved     licenses: [ [Object] ],
32 silly install resolved     tags: [ 'syslog', 'system', 'log', 'logging' ],
32 silly install resolved     gypfile: true,
32 silly install resolved     gitHead: '30d445559617d439714418cdc1e847efbb49c5a0',
32 silly install resolved     readme: '# Node-Syslog\n\nv1.1.7\n\nThis is a node module (add-on) to work with syslog (system log) daemon on unix systems.\nThis module has been tested with:\n\n* node.js v0.8.16\n* version v0.9.3\n* metalog\n* syslog-ng 3.1.1\n\nRead the [setMask wiki page](https://github.com/schamane/node-syslog/wiki/setMask) for using the `setMask` functionality.\n\nThe current version is compatible to node 0.8.x and higher. For older node.js versions, please use node-syslog v1.0.3\n\nNode-syslog does not officially support Darwin OS and MS Windows but should work fine.\n\n## Authors\n\n*   Nazar Kulyk\n*   Jeremy Childs\n*   Sam Roberts\n*   Ben Noordhuis\n\n## Installation\n\n### npm\n\n      npm install node-syslog\n\n### manual\n\n      git clone\n      node-gyp configure build\n\n## Usage\n\nFor more information about how to use module check test.js\n\n     var Syslog = require(\'node-syslog\');\n\n     Syslog.init("node-syslog", Syslog.LOG_PID | Syslog.LOG_ODELAY, Syslog.LOG_LOCAL0);\n     Syslog.log(Syslog.LOG_INFO, "Node Syslog Module output " + new Date());\n     Syslog.close();\n\nCheck your /var/log/messages (syslog, syslog-ng), or /var/log/everything/current (metalog) file for any test entry.\n',
32 silly install resolved     readmeFilename: 'README.md',
32 silly install resolved     _id: 'node-syslog@1.1.7',
32 silly install resolved     _shasum: '946304158f5ef13b98ee6694acfa0f8028747b35',
32 silly install resolved     _from: 'git://github.com/schamane/node-syslog',
32 silly install resolved     _resolved: 'git://github.com/schamane/node-syslog#30d445559617d439714418cdc1e847efbb49c5a0',
32 silly install resolved     _fromGithub: true } ]
33 info install node-syslog@1.1.7 into /Users/mikecostello/Documents/git-projects
34 info installOne node-syslog@1.1.7
35 verbose installOne of node-syslog to /Users/mikecostello/Documents/git-projects not in flight; installing
36 verbose lock using /Users/mikecostello/.npm/_locks/node-syslog-a2a67a2689b40d59.lock for /Users/mikecostello/Documents/git-projects/node_modules/node-syslog
37 silly install write writing node-syslog 1.1.7 to /Users/mikecostello/Documents/git-projects/node_modules/node-syslog
38 silly cache addNamed cb node-syslog@1.1.7
39 verbose unbuild node_modules/node-syslog
40 verbose gentlyRm vacuuming /Users/mikecostello/Documents/git-projects/node_modules/node-syslog
41 verbose tar unpack /Users/mikecostello/.npm/node-syslog/1.1.7/package.tgz
42 verbose tar unpacking to /Users/mikecostello/Documents/git-projects/node_modules/node-syslog
43 verbose gentlyRm vacuuming /Users/mikecostello/Documents/git-projects/node_modules/node-syslog
44 silly gunzTarPerm modes [ '755', '644' ]
45 silly gunzTarPerm extractEntry package.json
46 silly gunzTarPerm extractEntry .npmignore
47 silly gunzTarPerm extractEntry README.md
48 silly gunzTarPerm extractEntry node-syslog.js
49 silly gunzTarPerm extractEntry test.js
50 silly gunzTarPerm extractEntry compat-inl.h
51 silly gunzTarPerm extractEntry Changelog.md
52 silly gunzTarPerm extractEntry node-syslog.h
53 silly gunzTarPerm extractEntry binding.gyp
54 silly gunzTarPerm extractEntry LICENSE.txt
55 silly gunzTarPerm extractEntry syslog.cc
56 silly gunzTarPerm extractEntry compat.h
57 info preinstall node-syslog@1.1.7
58 silly install resolved []
59 verbose about to build /Users/mikecostello/Documents/git-projects/node_modules/node-syslog
60 info build /Users/mikecostello/Documents/git-projects/node_modules/node-syslog
61 verbose linkStuff [ false,
61 verbose linkStuff   false,
61 verbose linkStuff   false,
61 verbose linkStuff   '/Users/mikecostello/Documents/git-projects/node_modules' ]
62 info linkStuff node-syslog@1.1.7
63 verbose linkBins node-syslog@1.1.7
64 verbose linkMans node-syslog@1.1.7
65 verbose rebuildBundles node-syslog@1.1.7
66 info install node-syslog@1.1.7
67 verbose unsafe-perm in lifecycle true
68 info node-syslog@1.1.7 Failed to exec install script
69 verbose unlock done using /Users/mikecostello/.npm/_locks/node-syslog-a2a67a2689b40d59.lock for /Users/mikecostello/Documents/git-projects/node_modules/node-syslog
70 verbose stack Error: node-syslog@1.1.7 install: `node-gyp rebuild`
70 verbose stack Exit status 1
70 verbose stack     at EventEmitter.<anonymous> (/Users/mikecostello/.nvm/versions/io.js/v1.5.1/lib/node_modules/npm/lib/utils/lifecycle.js:213:16)
70 verbose stack     at emitTwo (events.js:87:13)
70 verbose stack     at EventEmitter.emit (events.js:169:7)
70 verbose stack     at ChildProcess.<anonymous> (/Users/mikecostello/.nvm/versions/io.js/v1.5.1/lib/node_modules/npm/lib/utils/spawn.js:14:12)
70 verbose stack     at emitTwo (events.js:87:13)
70 verbose stack     at ChildProcess.emit (events.js:169:7)
70 verbose stack     at maybeClose (child_process.js:984:16)
70 verbose stack     at Process.ChildProcess._handle.onexit (child_process.js:1057:5)
71 verbose pkgid node-syslog@1.1.7
72 verbose cwd /Users/mikecostello/Documents/git-projects
73 error Darwin 14.1.0
74 error argv "/Users/mikecostello/.nvm/versions/io.js/v1.5.1/bin/iojs" "/Users/mikecostello/.nvm/versions/io.js/v1.5.1/bin/npm" "install" "schamane/node-syslog"
75 error node v1.5.1
76 error npm  v2.7.0
77 error code ELIFECYCLE
78 error node-syslog@1.1.7 install: `node-gyp rebuild`
78 error Exit status 1
79 error Failed at the node-syslog@1.1.7 install script 'node-gyp rebuild'.
79 error This is most likely a problem with the node-syslog package,
79 error not with npm itself.
79 error Tell the author that this fails on your system:
79 error     node-gyp rebuild
79 error You can get their info via:
79 error     npm owner ls node-syslog
79 error There is likely additional logging output above.
80 verbose exit [ 1, true ]
81 verbose unbuild node_modules/node-syslog
82 info preuninstall node-syslog@1.1.7
83 info uninstall node-syslog@1.1.7
84 verbose unbuild rmStuff node-syslog@1.1.7 from /Users/mikecostello/Documents/git-projects/node_modules
85 info postuninstall node-syslog@1.1.7
86 verbose gentlyRm vacuuming /Users/mikecostello/Documents/git-projects/node_modules/node-syslog
87 silly gentlyRm purging /Users/mikecostello/Documents/git-projects/node_modules/node-syslog
88 silly gentlyRm removing /Users/mikecostello/Documents/git-projects/node_modules
89 silly gentlyRm finished vacuuming up to /Users/mikecostello/Documents/git-projects
```

**Comment by sam-github** [comment]
@MikeCostello I'm almost complete re-writing with nan. Actually, its complete, but I'm not happy with the buffer conversions, and Ben had some suggestions for making it better: https://github.com/sam-github/modern-syslog/pull/1

I hope to finish this week, but its looking to be quite busy.

**Comment by MikeCostello** [comment]
Thank you @sam-github @bnoordhuis! It will probably help out https://github.com/etsy/statsd/commit/4a555df3c4f8abde146d154408f856423b049f1f too.

**Comment by DaAwesomeP** [comment]
Is it finished? I am getting an error too:
```
0 info it worked if it ends with ok
1 verbose cli [ '/usr/bin/iojs', '/usr/bin/npm', 'install', '-g', 'node-syslog' ]
2 info using npm@2.8.3
3 info using node@v1.8.1
4 verbose install initial load of /usr/lib/package.json
5 verbose readDependencies loading dependencies from /usr/lib/package.json
6 silly cache add args [ 'node-syslog', null ]
7 verbose cache add spec node-syslog
8 silly cache add parsed spec { raw: 'node-syslog',
8 silly cache add   scope: null,
8 silly cache add   name: 'node-syslog',
8 silly cache add   rawSpec: '',
8 silly cache add   spec: '*',
8 silly cache add   type: 'range' }
9 silly addNamed node-syslog@*
10 verbose addNamed "*" is a valid semver range for node-syslog
11 silly addNameRange { name: 'node-syslog', range: '*', hasData: false }
12 silly mapToRegistry name node-syslog
13 silly mapToRegistry using default registry
14 silly mapToRegistry registry https://registry.npmjs.org/
15 silly mapToRegistry uri https://registry.npmjs.org/node-syslog
16 verbose addNameRange registry:https://registry.npmjs.org/node-syslog not in flight; fetching
17 verbose request uri https://registry.npmjs.org/node-syslog
18 verbose request no auth needed
19 info attempt registry request try #1 at 16:04:06
20 verbose request id dafaf04183b381e8
21 verbose etag "DJ3U77DXPEYVRPFJ0UJDUEY9Z"
22 http request GET https://registry.npmjs.org/node-syslog
23 http 304 https://registry.npmjs.org/node-syslog
24 silly get cb [ 304,
24 silly get   { date: 'Sat, 09 May 2015 21:04:06 GMT',
24 silly get     via: '1.1 varnish',
24 silly get     'cache-control': 'max-age=60',
24 silly get     etag: '"DJ3U77DXPEYVRPFJ0UJDUEY9Z"',
24 silly get     age: '0',
24 silly get     connection: 'keep-alive',
24 silly get     'x-served-by': 'cache-ord1731-ORD',
24 silly get     'x-cache': 'HIT',
24 silly get     'x-cache-hits': '1',
24 silly get     'x-timer': 'S1431205446.909621,VS0,VE38',
24 silly get     vary: 'Accept' } ]
25 verbose etag https://registry.npmjs.org/node-syslog from cache
26 verbose get saving node-syslog to /root/.npm/registry.npmjs.org/node-syslog/.cache.json
27 silly addNameRange number 2 { name: 'node-syslog', range: '*', hasData: true }
28 silly addNameRange versions [ 'node-syslog',
28 silly addNameRange   [ '0.6.0',
28 silly addNameRange     '0.6.1',
28 silly addNameRange     '0.6.2',
28 silly addNameRange     '1.0.0',
28 silly addNameRange     '1.0.2',
28 silly addNameRange     '1.1.0',
28 silly addNameRange     '1.1.1',
28 silly addNameRange     '1.1.2',
28 silly addNameRange     '1.1.3',
28 silly addNameRange     '1.0.3',
28 silly addNameRange     '1.1.5',
28 silly addNameRange     '1.1.6',
28 silly addNameRange     '1.1.7',
28 silly addNameRange     '1.2.0' ] ]
29 silly addNamed node-syslog@1.2.0
30 verbose addNamed "1.2.0" is a plain semver version for node-syslog
31 silly cache afterAdd node-syslog@1.2.0
32 verbose afterAdd /root/.npm/node-syslog/1.2.0/package/package.json not in flight; writing
33 verbose afterAdd /root/.npm/node-syslog/1.2.0/package/package.json written
34 silly install resolved [ { name: 'node-syslog',
34 silly install resolved     version: '1.2.0',
34 silly install resolved     main: './node-syslog',
34 silly install resolved     engines: { node: '>=0.8.0' },
34 silly install resolved     os: [ '!win32' ],
34 silly install resolved     scripts: { test: 'node test.js', install: 'node-gyp rebuild' },
34 silly install resolved     description: 'Node module to support sending messages to syslog daemon',
34 silly install resolved     author: { name: 'Nazar Kulyk', email: 'nasar.kulyk@googlemail.com' },
34 silly install resolved     homepage: 'http://github.com/schamane/node-syslog',
34 silly install resolved     repository:
34 silly install resolved      { type: 'git',
34 silly install resolved        url: 'git+ssh://git@github.com/schamane/node-syslog.git' },
34 silly install resolved     bugs: { url: 'https://github.com/schamane/node-syslog/issues' },
34 silly install resolved     licenses: [ [Object] ],
34 silly install resolved     tags: [ 'syslog', 'system', 'log', 'logging' ],
34 silly install resolved     gypfile: true,
34 silly install resolved     gitHead: 'fb8fe83564ab2338fc6edb7d335be63a13bcae1e',
34 silly install resolved     _id: 'node-syslog@1.2.0',
34 silly install resolved     _shasum: 'ecd4dbe5a83668392ce1d6dc747010f9df1f503a',
34 silly install resolved     _from: 'node-syslog@*',
34 silly install resolved     _npmVersion: '1.4.21',
34 silly install resolved     _npmUser: { name: 'schamane', email: 'nasar.kulyk@googlemail.com' },
34 silly install resolved     maintainers: [ [Object] ],
34 silly install resolved     dist:
34 silly install resolved      { shasum: 'ecd4dbe5a83668392ce1d6dc747010f9df1f503a',
34 silly install resolved        tarball: 'http://registry.npmjs.org/node-syslog/-/node-syslog-1.2.0.tgz' },
34 silly install resolved     directories: {},
34 silly install resolved     _resolved: 'https://registry.npmjs.org/node-syslog/-/node-syslog-1.2.0.tgz',
34 silly install resolved     readme: 'ERROR: No README data found!' } ]
35 info install node-syslog@1.2.0 into /usr/lib
36 info installOne node-syslog@1.2.0
37 verbose installOne of node-syslog to /usr/lib not in flight; installing
38 verbose lock using /root/.npm/_locks/node-syslog-710c6e731352c788.lock for /usr/lib/node_modules/node-syslog
39 silly install write writing node-syslog 1.2.0 to /usr/lib/node_modules/node-syslog
40 verbose unbuild lib/node_modules/node-syslog
41 silly gentlyRm /usr/lib/node_modules/node-syslog is being purged from base /usr
42 verbose gentlyRm don't care about contents; nuking /usr/lib/node_modules/node-syslog
43 verbose tar unpack /root/.npm/node-syslog/1.2.0/package.tgz
44 verbose tar unpacking to /usr/lib/node_modules/node-syslog
45 silly gentlyRm /usr/lib/node_modules/node-syslog is being purged
46 verbose gentlyRm don't care about contents; nuking /usr/lib/node_modules/node-syslog
47 silly gunzTarPerm modes [ '755', '644' ]
48 silly gunzTarPerm extractEntry package.json
49 silly gunzTarPerm extractEntry .npmignore
50 silly gunzTarPerm extractEntry README.md
51 silly gunzTarPerm extractEntry node-syslog.js
52 silly gunzTarPerm extractEntry test.js
53 silly gunzTarPerm extractEntry compat-inl.h
54 silly gunzTarPerm extractEntry Changelog.md
55 silly gunzTarPerm extractEntry node-syslog.h
56 silly gunzTarPerm extractEntry binding.gyp
57 silly gunzTarPerm extractEntry LICENSE.txt
58 silly gunzTarPerm extractEntry syslog.cc
59 silly gunzTarPerm extractEntry compat.h
60 verbose write writing to /usr/lib/node_modules/node-syslog/package.json
61 info preinstall node-syslog@1.2.0
62 verbose readDependencies loading dependencies from /usr/lib/node_modules/node-syslog/package.json
63 verbose readDependencies loading dependencies from /usr/lib/node_modules/node-syslog/package.json
64 silly install resolved []
65 verbose about to build /usr/lib/node_modules/node-syslog
66 info build /usr/lib/node_modules/node-syslog
67 info linkStuff node-syslog@1.2.0
68 silly linkStuff node-syslog@1.2.0 has /usr/lib/node_modules as its parent node_modules
69 silly linkStuff node-syslog@1.2.0 is part of a global install
70 silly linkStuff node-syslog@1.2.0 is installed into a global node_modules
71 silly linkStuff node-syslog@1.2.0 is installed into the top-level global node_modules
72 verbose linkBins node-syslog@1.2.0
73 verbose linkMans node-syslog@1.2.0
74 verbose rebuildBundles node-syslog@1.2.0
75 info install node-syslog@1.2.0
76 verbose unsafe-perm in lifecycle false
77 info node-syslog@1.2.0 Failed to exec install script
78 verbose unlock done using /root/.npm/_locks/node-syslog-710c6e731352c788.lock for /usr/lib/node_modules/node-syslog
79 verbose stack Error: node-syslog@1.2.0 install: `node-gyp rebuild`
79 verbose stack Exit status 1
79 verbose stack     at EventEmitter.<anonymous> (/usr/lib/node_modules/npm/lib/utils/lifecycle.js:213:16)
79 verbose stack     at emitTwo (events.js:87:13)
79 verbose stack     at EventEmitter.emit (events.js:169:7)
79 verbose stack     at ChildProcess.<anonymous> (/usr/lib/node_modules/npm/lib/utils/spawn.js:24:14)
79 verbose stack     at emitTwo (events.js:87:13)
79 verbose stack     at ChildProcess.emit (events.js:169:7)
79 verbose stack     at maybeClose (child_process.js:953:16)
79 verbose stack     at Process.ChildProcess._handle.onexit (child_process.js:1020:5)
80 verbose pkgid node-syslog@1.2.0
81 verbose cwd /home/vm
82 error Linux 3.16.0-4-amd64
83 error argv "/usr/bin/iojs" "/usr/bin/npm" "install" "-g" "node-syslog"
84 error node v1.8.1
85 error npm  v2.8.3
86 error code ELIFECYCLE
87 error node-syslog@1.2.0 install: `node-gyp rebuild`
87 error Exit status 1
88 error Failed at the node-syslog@1.2.0 install script 'node-gyp rebuild'.
88 error This is most likely a problem with the node-syslog package,
88 error not with npm itself.
88 error Tell the author that this fails on your system:
88 error     node-gyp rebuild
88 error You can get their info via:
88 error     npm owner ls node-syslog
88 error There is likely additional logging output above.
89 verbose exit [ 1, true ]
90 verbose unbuild lib/node_modules/node-syslog
91 info preuninstall node-syslog@1.2.0
92 info uninstall node-syslog@1.2.0
93 verbose unbuild rmStuff node-syslog@1.2.0 from /usr/lib/node_modules
94 info postuninstall node-syslog@1.2.0
95 silly gentlyRm /usr/lib/node_modules/node-syslog is being purged from base /usr
96 verbose gentlyRm don't care about contents; nuking /usr/lib/node_modules/node-syslog
97 silly vacuum-fs purging /usr/lib/node_modules/node-syslog
98 silly vacuum-fs quitting because other entries in /usr/lib/node_modules
```
